### ISSUE TITLE
An issue that is frustrating

### ISSUE THREAD

**Issue by ghost** [haxball/haxball-issues#1460]
Extrapolation resets itself during the match - I can feel it from the character's movement. When I press the ESC button twice(return to menu and back to match screen), it goes back to setting I adjusted. Is there a way to prevent extrapolation resetting itself during the game? It really distorts the gameplay. I also wonder If this occurs for anyone else.
Windows 10
Tried on different browsers.

**Comment by thenorthstar** [comment]
> Extrapolation resets itself during the match - I can feel it from the character's movement. When I press the ESC button twice(return to menu and back to match screen), it goes back to setting I adjusted. Is there a way to prevent extrapolation resetting itself during the game? It really distorts the gameplay. I also wonder If this occurs for anyone else.
> Windows 10
> Tried on different browsers.

@gdnzz Yes, it was happening on also me, about a couple of months ago. It was usually happening when I changed the status of the screen (full screen or window) or made adjustment on sound or lighting of my computer. After doing it, I was playing like under zero extrapolation. Moreover, in contrary of your situation; my extrapolation was not being fixed when I pressed ESC twice. It was being fixed itself after a short time.

Now, frankly I wasn't suffer from this situation for a long time (about 2 months I guess). Probably because of spending less time in game. Also, please try it in different rooms. For example, I had got suffered from it a lot, when I was playing in racing rooms (former DJ racing rooms, maybe some people here know it) whereas had not got suffered from it when playing in the most of the rest.
